gradle/versions.gradle

```diff
@@ -23,6 +23,11 @@ dependencyManagement {
     dependency 'com.google.guava:guava:32.0.1-jre'
 
     dependency 'com.squareup.okhttp3:okhttp:4.11.0'
+    /*
+    com.squareup.okhttp3:logging-interceptor:4.9.0 // CVE-2023-0833
+      \--- org.web3j:core:4.10.2
+    */
+    dependency 'com.squareup.okhttp3:logging-interceptor:4.11.0'
```

Comment:
This is still necessary despite web3j upgrade?

Reply:
unfortunately yes ... the fixed version of this library is available as part of web3j latest version ... however, web3j latest version is not compatible with ethsigner anymore due to dependency on signers library. (one of the exception package has changed that signers module is using from web3j).

Reply:
yes...